src/handlers/comment/action.ts

```diff
@@ -28,13 +29,21 @@ export const handleComment = async (): Promise<void> => {
     const userCommand = allCommands.find((i) => i.id == command);
 
     if (userCommand) {
-      const { handler, callback, successComment, failureComment } = userCommand;
+      const { id, handler, callback, successComment, failureComment } = userCommand;
       logger.info(`Running a comment handler: ${handler.name}`);
 
       const { payload: _payload } = getBotContext();
       const issue = (_payload as Payload).issue;
       if (!issue) continue;
 
+      const feature = config.command.find((e) => e.name === id.split("/")[1]);
+
+      if (!feature?.enabled && id !== "/help") {
```

Comment:
Not a good way to use hardcoded `/help` which has been defined in the command list.

Reply:
you might need to figure out to resolve conversations.

Reply:
> you might need to figure out to resolve conversations.

Figured it out

Reply:
You need to post the proof before marking this conversation resolved.

Reply:
I pushed a commit and then resolved em